interface/resources/qml/hifi/dialogs/TabletAboutDialog.qml

```diff
@@ -14,7 +14,7 @@ import stylesUit 1.0
 
 Rectangle {
     width: 480
-    height: 706
+    height: 950
```

Comment:
Have you checked the dialog display on the tablet?

Reply:
I checked in VR and it did appear to be off the screen, dunno why it doesn't scroll. I guess that QML page wasn't setup to trigger that. I tightened spacing on the whole page so it would fit.